scripts/exgdas_global_marine_analysis_prep.py

```diff
@@ -507,12 +507,12 @@ def find_clim_ens(input_date):
 # Copy initial condition
 ics_list = []
 # ocean IC's
-mom_ic_src = glob.glob(os.path.join(bkg_dir, f'{RUN}.*.ocnf003.nc'))[0]
```

Comment:
are you defining `$GDUMP` anywhere?

Reply:
Yes @CoryMartin-NOAA , in the jjob.

Reply:
ok sure, but then don't you need something like line 126?
https://github.com/NOAA-EMC/GDASApp/blob/61aa906e8e7bf68dd3db54a1e2d5f39a8b6f0224/scripts/exgdas_global_marine_analysis_prep.py#L126

Reply:
Ha!!!